app/src/main/java/com/ayush/weatherapp/home/HomeActivity.java

```diff
@@ -126,6 +129,10 @@
     }
   }
 
+  @Override public void setHomeBackground(Drawable viewBackground) {
```

Comment:
presenter should not send the drawable. It could send the drawable id from which the view should display it.

Reply:
:ok: